ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -51,19 +51,23 @@ SettingsPageLayout {
 
     signal signMintTransactionOpened(int chainId, string accountAddress)
 
-    function setFeeLoading() {
-        root.isFeeLoading = true
-        root.feeText = ""
-        root.errorText = ""
-    }
-
     signal remoteSelfDestructCollectibles(var holdersModel,
                                           int chainId,
                                           string accountName,
                                           string accountAddress)
 
     signal signSelfDestructTransactionOpened(int chainId)
 
+    signal airdropClicked()
```

Comment:
This signal seems to be used only to pass information between instances of internally defined components. Could we move it to `d` and keep as an implementation detail?